doc/site/core/IO.markdown

```diff
@@ -0,0 +1,34 @@
+^title IO Class
```

Comment:
Do you mind making the file name lowercase ("io.html") to match the other files?